LFGBulletinBoard/LibGPIMinimapButton.lua

```diff
@@ -11,36 +17,43 @@ local function BottomZoom(button)
 	button.Lib_GPI_MinimapButton.icon:SetTexCoord( deltaX, 1 - deltaX,  deltaY, 1 - deltaY)
 end
 
+-- Calculates the the angle and distance of the minimap button based on cursor position about the minimap center.
+---@param button MinimapButton
 local function onUpdate(button)
 	local mx, my = Minimap:GetCenter()
 	local px, py = GetCursorPosition()
-	local w=((Minimap:GetWidth() / 2) + 5)
 	local scale = Minimap:GetEffectiveScale()
-	px, py = px / scale, py / scale
-	local dx,dy= px - mx,py - my
-	local dist=math.sqrt(dx*dx+dy*dy)/w
```

Comment:
moved this math into the conditional to only be used when necessary (`onUpdate` functions run every frame)